pkg/virt-handler/virtwrap/api/defaults.go

```diff
@@ -0,0 +1,54 @@
+package api
+
+func SetDefaults_Devices(devices *Devices) {
+	// Add mandatory spice device
+	devices.Graphics = []Graphics{
```

Comment:
Annotations might be a solution here:

```yaml
metadata:
  name: myvm
  annotations:
    "kubevirt.io/spice": "no",
    "kubevirt.io/vnc": "no",
    "kubevirt.io/console": "no",
```
For opting out of infra-devices that might be a good solution. However it largely depends on how much we will support arbitrary configurations in these areas.

Reply:
Why is a graphics device mandatory at all?

Reply:
I actually wonder - could you provide an example with an extensive features section?

Just to get an impression if it will be a bucket for everything … :)